quil-rs/src/instruction/mod.rs

```diff
@@ -502,7 +502,7 @@ impl Instruction {
                         set.insert(qubit);
                         set
                     }
-                    None => qubits_available,
+                    None => qubits_available.iter().collect(),
```

Comment:
The function this is in used to take a `HashSet<&Qubit>` as input and output. Since it now takes a `& HashSet<Qubit>` as input, we have to convert it to `HashSet<&Qubit>` in the one case we want to return that set.